Dangerfile

```diff
@@ -2,6 +2,11 @@
 require 'json'
 require 'active_support/core_ext/string/output_safety'
 
+warn 'This PR does not have any assignees yet.' unless github.pr_json['assignee']
+
+warn 'Please, set correct Jira link in the PR comment.' if 
+github.pr_body.include? "https://jira.ouroath.com/browse/OMSDK-"
```

Comment:
This case is always `true`, because `https://jira.ouroath.com/browse/OMSDK-` is part of PR template.

Reply:
Please, move danger improvements into separate PRs.

Reply:
Ok